Content.Server/Chemistry/ReagentEffects/AdjustAlert.cs

```diff
@@ -13,9 +13,21 @@ public sealed partial class AdjustAlert : ReagentEffect
     [DataField]
     public bool Clear;
 
+    /// <summary>
+    /// Show cooldown progress over the alert
+    /// </summary>
     [DataField]
     public bool Cooldown;
 
+    /// <summary>
+    /// Automatically remove the alert after a set time
+    /// </summary>
+    [DataField]
+    public bool AutoRemove;
+
+    /// <summary>
+    /// The length of the cooldown or the delay before autoRemove (in seconds) .
+    /// </summary>
     [DataField]
     public float Time;
```

Comment:
Why don't you just use the time of the cooldown and hide the duration instead? That way we don't have 2 separate code-paths doing identical things and all the difference is is whether the duration graphic is ticking down.

Reply:
Probably simpler if I just show what I mean. Pushed the commit, can always just revert that one